images/base/ubuntu.Dockerfile

```
@@ -30,16 +30,22 @@ RUN apt-get update && \
     pipx \
     python3 \
     python3-pip \
-    software-properties-common \
+# software-properties-common \
     sudo \
     systemd \
     systemd-sysv \
     unzip \
     vim \
     wget \
-    rsync && \
+    rsync
+
```

```suggestion
   rsync && \
   rm -rf /var/lib/apt/lists/*
```
This will reduce this layers size.